### PR TITLE
Add MG5 EWKZ2Jets_ZToQQ

### PR DESCRIPTION
**Pull request by jennetd**
I am working on an EWK Z sample with hadronic Z decay following the example here for Z(ll): 
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkzlljj_5f_LO

I plan to submit similar samples for EWK W+ and W- after this one has been reviewed.  Could you please advise if you have any suggested changes? Thanks.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/EWKZ2Jets_ZToQQ/.log [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Starting job on  Tue Jan 18 14:07:20 CST 2022
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/EWKZ2Jets_ZToQQ/.log:1]: I assume the .log was committed by mistake?

Comment (jennetd) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/EWKZ2Jets_ZToQQ/.log:1]: Yes. I will remove it

**Comment by agrohsje** [comment]
Looks already quite good. Question: do you use drjj in the analysis, or mjj cuts? I see them at 0.4 and 10 GeV?

**Comment by jennetd** [comment]
> Looks already quite good. Question: do you use drjj in the analysis, or mjj cuts? I see them at 0.4 and 10 GeV?

We apply an mjj cut and delta eta cut, but would like this sample to be useful for another analysis as well. I think we can use the minimum cuts that you recommend for these vars.

**Comment by jennetd** [comment]
@agrohsje Do you recommend reducing these cut values or leaving as is?

**Comment by agrohsje** [comment]
What is your mjj cut at analysis level?

**Comment by jennetd** [comment]
Hi Alexander, the min mjj is 250 GeV. However, in the resolved analysis this cut is not applied to the Z candidate. Is there a way to apply the mjj cut to only the VBF jets? Thanks.

**Comment by agrohsje** [comment]
Setting cut_decays to false should leave the particles from the decay out so that the mjj cut is only applied to the VBF jets. I would suggest setting jet dr jj to 0.1 and mjj to 200. @sihyunjeon  what do you think? Once agreed this is ok to merge from my side.

**Comment by sihyunjeon** [comment]
Hello, as Alexander said if you turn off "cut_decays" then the jets from Z will not be applied with cuts.
So as Alexander suggested, I edited drjj to 0.1 and mjj to 200 GeV and turned off cut_decays. Will merge.

**Comment by jennetd** [comment]
Great, thank you very much!

**Pull request merged**